clientv3/watch.go

```diff
@@ -25,12 +25,17 @@ import (
 	pb "go.etcd.io/etcd/etcdserver/etcdserverpb"
 	mvccpb "go.etcd.io/etcd/mvcc/mvccpb"
 
+	"github.com/coreos/pkg/capnslog"
```

Comment:
We are deprecating this logger in next release (and current master branch). Let's pass clientv3's logger to watcher and use that.
cc @gyuho

Reply:
Looking at init() in clientv3/client.go :
```
		lg, err = lcfg.Build() // info level logging
```
Did you mean that the above lg should be passed to watch.go ?

I am still trying to find out how to pass the lg to func's in watch.go .

Reply:
Yes. That is what I meant. I think it can be passed in creating newWatcher:
https://github.com/etcd-io/etcd/blob/eaa0612e0260a6ec16aa3be1f6ced21f059dd794/clientv3/watch.go#L234

Reply:
@jingyih 
Your comment has been addressed.